lib/utils.js

```diff
@@ -13,16 +15,19 @@ const internals = {};
 
 module.exports.log = function (err, options, callback) {
 
-    const log = Fs.createWriteStream(options.logPath, options.writeStreamOptions);
-    const formattedErr = {
-        message: err.message,
-        stack: err.stack,
-        timestamp: Date.now()
-    };
+    Mkdirp(Path.dirname(options.logPath), () => {
```

Comment:
Why aren't we handling the error that could be returned from `mkdirp`? I would expect that if we have an error with `mkdirp` that we would propagate that to the callback.